### PR TITLE
chore: adjust ckb to eth fee

### PR DESCRIPTION
**Pull request by huwenchao**

### CODE REVIEW

File: configs/mainnet-asset-white-list.json [hunk 1]
```diff
@@ -5,414 +5,382 @@
...
+    }
   },
   {
     "address": "0x56d811088235F11C8920698a204A5010a788f4b3",
     "name": "BZRX",
     "symbol": "BZRX",
     "logoURI": "https://cryptologos.cc/logos/bzx-protocol-bzrx-logo.svg?v=013",
     "decimal": 18,
-    "minimalBridgeAmount": "343000000000000000000",
+    "minimalBridgeAmount": "Inf00000",
```
Comment (mkxbl) [inline configs/mainnet-asset-white-list.json:284]: Seems not right here?

File: configs/mainnet-asset-white-list.json [hunk 2]
```diff
@@ -5,414 +5,382 @@
     "symbol": "ETH",
     "logoURI": "https://cryptologos.cc/logos/ethereum-eth-logo.svg?v=002",
     "decimal": 18,
-    "minimalBridgeAmount": "30000000000000000",
+    "minimalBridgeAmount": "45000000000000000",
     "bridgeFee": {
-      "in": "1590000000000000",
-      "out": "15000000000000000"
-    },
-    "sudtArgs": "0x9657b32fcdc463e13ec9205914fd91c443822a949937ae94add9869e7f2e1de8"
```
Comment (mkxbl) [inline configs/mainnet-asset-white-list.json:13]: remove `sudtArgs` field?

File: configs/mainnet-asset-white-list.json (hunk 1 above)
Comment (huwenchao) [reply configs/mainnet-asset-white-list.json:284]: The weightedAvgPrice returned from API is 0, I'll add a exception handle here.

File: configs/mainnet-asset-white-list.json (hunk 2 above)
Comment (huwenchao) [reply configs/mainnet-asset-white-list.json:13]: `sudtArgs` is not used in the config, just listed here to show.
The show function of this file have been moved to `all-bridged-tokens.json`.

**Pull request merged**